roles/ood_user_reg_cloud/tasks/main.yml

```diff
@@ -1,48 +1,82 @@
 ---
-- name: Enable Copr Repos
-  shell: yum -y copr enable "{{ item.repo_name }}"
-  with_items: "{{ copr_repos }}"
-  when: enable_copr == true and inventory_hostname in item.host
-
-- name: Install mod_wsgi for httpd24
-  yum: name="{{ mod_wsgi_pkg_name }}" state=latest
-
-- name: Install pip via yum
-  yum: name=python3-pip state=latest
-
-- name: Install virtualenv via pip
-  pip: name=virtualenv state=latest
+- name: Install packages via yum
+  yum:
+    name:
+      - python3-pip
+      - python3-devel
+    state: latest
 
 - name: Remove existing user register app install
   file:
     path: "{{ user_register_app_path }}"
     state: absent
 
-- name: Creates directory to clone the user register app
-  file:
-    path: "{{ user_register_app_path }}"
-    state: directory
-
 - name: Clone user register app form from gitlab
   git:
     repo: "{{ user_register_app_repo }}"
     dest: "{{ user_register_app_path }}"
 
+- name: Change ownership of directory
+  file:
+    path: "{{ user_register_app_path }}"
+    owner: "{{ RegUser_app_user }}"
+    group: apache
+    state: directory
+    recurse: yes
+
 - name: Install requirements in virtualenv
   pip:
     requirements: requirements.txt
     virtualenv: venv
+    virtualenv_command: /usr/bin/python3 -m venv
+    chdir: "{{ user_register_app_path }}"
+
+- name: Install uwsgi in virtualenv
+  pip:
+    name: uwsgi
+    virtualenv: venv
+    virtualenv_command: /usr/bin/python3 -m venv
     chdir: "{{ user_register_app_path }}"
 
+- name: Create log directory
+  file:
+    path: "{{ user_register_app_path }}/log"
+    owner: "{{ RegUser_app_user }}"
+    group: apache
+    state: directory
+
+- name: Copy Celery service template
+  template:
+    src: "{{ item }}"
+    dest: "/etc/systemd/system/{{ item }}"
+  with_items:
+    - celery.service
+
+- name: Start ood_account.service
```

Comment:
Change name of task to not use ood_ tag.  This runs on ohpc but really should be named for the task not the location.